src/ComposedMap.cpp

```diff
@@ -212,19 +212,36 @@ void ComposedMap<MemorySpace>::WrapCoeffs(Kokkos::View<double*, Kokkos::HostSpac
 }
 
 #if defined(MPART_ENABLE_GPU)
-template<typename MemorySpace>
-void ComposedMap<MemorySpace>::SetCoeffs(Kokkos::View<const double*, Kokkos::DefaultExecutionSpace::memory_space> coeffs)
```

Comment:
This is a holdover from previous code: would you mind updating any time it says this "Kokkos::DefaultExecutionSpace::memory_space" and changing it to `mpart::DeviceSpace`?

Reply:
This allows us to choose our device in one singular location, particularly when the device we want is not the default.